cmd/neofs-cli/modules/container/set_eacl.go

```diff
@@ -106,16 +106,11 @@ Container ID in EACL table will be substituted with ID from the CLI.`,
 			cmd.Println("ACL extension is enabled in the container, continue processing.")
 		}
 
-		var setEACLPrm internalclient.SetEACLPrm
-		setEACLPrm.SetClient(cli)
-		setEACLPrm.SetTable(eaclTable)
-		setEACLPrm.SetPrivateKey(*pk)
-
+		var eaclPrm client.PrmContainerSetEACL
```

Comment:
var renaming could've been avoided

Reply:
Exactly what I tried to do, but failed with this one. Fixed.